### ISSUE TITLE
Failed uploads should be automatically retried

### ISSUE THREAD

**Issue by paolobarbolini** [haiwen/seahub#2081]
I think the resumable upload feature in seafile pro server is incomplete without the ability to automatically retry failed uploads.

When I upload lots of files trough the website sometimes a request would fail and the upload of that file would be skipped completely, meaning that at the end I have to go back and retry manually every file that failed by finding it on my computer and re dragging it in the web interface.

This is even worse when I send an upload link to another user who doesn't know this happens, because I usually get replies about the upload failing and I have to explain to them to retry the upload until it succeeds.

**Comment by shoeper** [comment]
Although I support this request it is technically not related to the resumable uploads feature.

**Comment by paolobarbolini** [comment]
Right, I was thinking about uploads of large files failing in the middle of the night, but this also applies to non resumable uploads.
